sockeye/utils.py

```diff
@@ -774,3 +774,17 @@ def grouper(iterable: Iterable, size: int) -> Iterable:
         if not chunk:
             return
         yield chunk
+
+
+def paste(*streams, add_eol=True) -> Iterable:
```

Comment:
can you add type information for the arguments?
It would also be helpful to have a unit test for this function.